framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -390,30 +390,37 @@ public S initialStore(UnderlyingAST underlyingAST, List<LocalVariableNode> param
     // the declared type instead of a refined type. Issue a warning to alert users?
     private void addInitialFieldValues(S store, ClassTree classTree, MethodTree methodTree) {
         boolean isConstructor = TreeUtils.isConstructor(methodTree);
+        boolean isStaticMethod =
+                ElementUtils.isStatic(TreeUtils.elementFromDeclaration(methodTree));
         TypeElement classEle = TreeUtils.elementFromDeclaration(classTree);
         for (FieldInitialValue<V> fieldInitialValue : analysis.getFieldInitialValues()) {
             VariableElement varEle = fieldInitialValue.fieldDecl.getField();
+            boolean fieldNotFromEnclosingClass = varEle.getEnclosingElement().equals(classEle);
```

Comment:
Should the `Not` be removed from the variable name?

Reply:
If `varEle.getEnclosingElement().equals(classEle)` is true, `varEle` is in `classEle`, then this field is not from enclosing class right?

Reply:
I'm not sure where the confusion is here. The enclosing element of `varEle` is some class. If that class is equal to `classEle`, then the field is directly within `classEle`.
What element do you mean with "the enclosing class"?

Reply:
Meaning `classEle` is the nested class. The enclosing class, encloses `classEle`

Reply:
It's usually easier to avoid negations. How about `isFieldOfCurrentClass`?

Reply:
Yes, I agree!